pennylane/wires.py

```diff
@@ -645,15 +648,15 @@ def __rsub__(self, other):
         return Wires((set(_process(other)) - set(self.labels)))
 
     def symmetric_difference(self, other):
-        """Return the symmetric difference of the current Wires object and either another Wires
-        object or an iterable that can be interpreted like a Wires object e.g., List.
+        """Return the symmetric difference of the current :class:`~.Wires` object and either another :class:`~.Wires`
+        object or an iterable that can be interpreted like a :class:`~.Wires` object e.g., ``List``.
 
         Args:
-            other (Any): Wires or any iterable that can be interpreted like a Wires object
-                to perform the union with. See _process for details on the interpretation.
+            other (Any): :class:`~.Wires` or any iterable that can be interpreted like a :class:`~.Wires` object
+                to perform the union with. See ``_process`` for details on the interpretation.
```

Comment:
See above re `_process`

```suggestion
                to perform the symmetric difference with. See ``_process`` for details on the interpretation.
```